banzai/munge.py

```diff
@@ -12,21 +12,18 @@ class SinistroModeNotSupported(Exception):
 
 
 def munge(image, pipeline_context):
-    telescope = dbs.get_telescope(image.telescope_id,
-                                  db_address=pipeline_context.db_address)
-
-    if 'sinistro' in telescope.camera_type.lower():
+    if 'sinistro' in image.telescope.camera_type.lower():
         if sinistro_mode_is_supported(image):
             munge_sinistro(image)
         else:
             raise SinistroModeNotSupported('Sinistro mode not supported {f}'.format(f=image.filename))
 
```

Comment:
Can we add here the NRES CRPIX1 & 2  fix? Maybe it is more appropriate in a second PR after this one,
but the following is required to get NRES frames through trim.trimmer.
```
    if image.header.get('CRPIX1') is None:
        image.header['CRPIX1'] = 0
    if image.header.get('CRPIX2') is None:
image.header['CRPIX2'] = 0
```